### ISSUE TITLE
Icon Request: Zodiac Signs

### ISSUE THREAD

**Issue by ghost** [FortAwesome/Font-Awesome#2831]
Would be much usefull to see zodiac signs

**Comment by qopel** [comment]
+1

**Comment by joaosaragossa** [comment]
+1

**Comment by Magidragon-Sapph** [comment]
+1

**Comment by thehappycoder** [comment]
+1

**Comment by rastarr** [comment]
+1

**Comment by kagge** [comment]
+1

**Comment by beaudavenport** [comment]
+1

**Comment by JoyceBabu** [comment]
+1

**Comment by sghribi** [comment]
+1

**Comment by wizysanjay** [comment]
+1

**Comment by borNfreee** [comment]
+1

**Comment by aoshmyanskaya** [comment]
+1

**Comment by alex21** [comment]
+1

**Comment by virtginia** [comment]
+1

**Comment by leecollings** [comment]
What's the news with this? Doesn't look like anything happened, and it's been over a year....

**Comment by pslcbs** [comment]
+1

**Comment by jacobrmcadam** [comment]
+1
Note there is a modern variant for capricorn that could be included as an alternative to the more classical symbols.
![cap-variant](https://cloud.githubusercontent.com/assets/10902570/11415809/85e1bfec-93bc-11e5-8034-aab0203d5565.jpg)

**Comment by Fazime** [comment]
+1

**Comment by codyn329** [comment]
+1

**Comment by Fazime** [comment]
+1 again

**Comment by jbnv** [comment]
+1.

**Comment by asialee6** [comment]
+1

**Comment by leecollings** [comment]
@tagliala What's going on with this? Are you waiting for a certain number of +1's?

**Comment by incredible-dev** [comment]
+1

**Comment by PeterShaggyNoble** [comment]
+1

**Comment by franverona** [comment]
I found this: http://www.alt-codes.net/zodiac-signs.php

It could be an alternative at the moment.

**Comment by leecollings** [comment]
I ended up getting hold of a free astrology font, and using that for symbols.

http://www.fontspace.com/category/astrology

**Comment by tagliala** [comment]
Note that this request is against our contributing guidelines:

https://github.com/FortAwesome/Font-Awesome/blob/master/CONTRIBUTING.md#requesting-new-icons

> Please do not request comprehensive sets of icons related to a very specific field.

I'm leaving this open but please consider to use fontawesome together with another set or a custom build

**Comment by leecollings** [comment]
What? Related to a very specific field.. you mean like all of the brand icons you have already? That guideline should not apply to Zodiac signs. It's the same as saying don't have a hospital icon because it relates to a specific field of medicine.

**Comment by tagliala** [comment]
>  It's the same as saying don't have a hospital icon because it relates to a specific field of medicine.

This is far from being true: hospital icon does not fall under "comprehensive set of icons", it is a request for an individual icon.

Take a look here: https://github.com/FortAwesome/Font-Awesome/wiki/Customize-Font-Awesome

> So, if you need a new icon in a short while, or you need a comprehensive set of icons **(e.g., a set of weather icons to build a forecast application) our suggestion is to customize Font Awesome**.

X-Ref: #8960

**Comment by leecollings** [comment]
> hospital icon does not fall under "comprehensive set of icons", it is a request for an individual icon.

So based on what you're saying, we can make individual requests for each sign, and that would be following the guidelines, like the hospital icon for example which has been approved and released?

**Comment by tagliala** [comment]
> So based on what you're saying, we can make individual requests for each sign, and that would be following the guidelines, like the hospital icon for example which has been approved and released?

No, a single request is fine because they are strictly related icons but you should consider a custom build for this use case

**Comment by leecollings** [comment]
I don't agree with you, and I think you are providing an excuse for not wanting to look into it. Double standards.

**Comment by jbnv** [comment]
> Please do not request comprehensive sets of icons related to a very specific field.

Paper, rock, scissors, lizard, Spock.

Battery status icons.

File type icons.

Payment icons.

Gender icons.

The precedent you have set clearly violates your own rules.

Speaking of the gender icons: Mars, Venus and Mercury are in the same "comprehensive sets of icons" as the zodiac icons. So you have _already_ started this set of icons.

**Comment by franverona** [comment]
I think any icon request will be related to an specific field, so I personally don't see any scenario were that rule will be ok.

Another thing is that FontAwesome team don't want to spend resources and time in adding these icons (completely understandable).

**Comment by tagliala** [comment]
> I think any icon request will be related to an specific field, so I personally don't see any scenario were that rule will be ok.

Unfortunately rules cannot cover any scenario. Zodiac signs are ok in a single requests, because they are 12 (cannot be more), strictly related and you need all of them.

Please let me quote from [Customize Font Awesome](https://github.com/FortAwesome/Font-Awesome/wiki/Customize-Font-Awesome) again:

> So, if you need a new icon in a short while, or **_you need a comprehensive set of icons (e.g., a set of weather icons to build a forecast application) our suggestion is to customize Font Awesome_**.

Is the above suggestion completely unacceptable?

Just a couple of examples:
* Before having some payment icons in the set, people used Font Awesome together with another font, [brandfont](http://www.rb.cm/brandfont/), you can take a [look at this comment](https://github.com/FortAwesome/Font-Awesome/issues/1009#issuecomment-53193234) from Dave
* [mfizz](http://fizzed.com/oss/font-mfizz) is a font set for technology icons because we have a lot of requests about languages and technologies which could not be implemented soon

> Another thing is that FontAwesome team don't want to spend resources and time in adding these icons (completely understandable).

Thanks for your understanding 👍.  You may not know that Dave is the only one to design new icons (it is a design choice). I also need to clarify that there is nothing against this specific request, which is open, labeled and treated like [all other requests](https://github.com/FortAwesome/Font-Awesome/issues?q=is%3Aopen+is%3Aissue+label%3A%22new+icon%22+sort%3Acomments-desc)

If you have any suggestion on how improve the [contributing guidelines for requesting new icons](https://github.com/FortAwesome/Font-Awesome/blob/master/CONTRIBUTING.md#requesting-new-icons) feel free to start a new issue

**Comment by Ladiczek** [comment]
+1

**Comment by leecollings** [comment]
I literally cannot believe how stubborn you are being over this. Considering how many users have shared their input and commenting about how they would like to see Zodiac signs, I think that clearly demonstrates that this is not just a very specific set of icons that will only be used for a small number of people.

I mean come on, you've got an icon for Yoast. How can this not be very specific? Yoast even get's it's own direct link out to their website from the detail page, which I find very biased and unfair.

I really fail to understand how a group of Zodiac icons, is different to the current group of Medical icons that are available. As previously mentioned, Medical Icons are very much specific to an individual subject or field. This is clearly going against your own terms.

Given the immense popularity for wanting these icons sets to be created, I find it incredibly disappointing that you're just not listening to what the people want, and instead think that your own flawed rules are better.

I'm going to create individual requests for each Zodiac icon shortly.

**Comment by jbnv** [comment]
> Before having some payment icons in the set, people used Font Awesome together with another font, brandfont, you can take a look at this comment from Dave.

But you made the payment icons anyway. 

If you really want to make an argument here, spin the specialized groups of icons into separate fonts. For even more credibility, open them up to the community for further development. 

But don't tell us "no requests of specialized icon sets; them's the rules" while you continue to add specialized sets of icons to the product.

**Comment by diziara** [comment]
...would this request now include a 13th icon from NASA's recent recalculation of the Zodiac?

**Comment by leecollings** [comment]
Any icons in this request would be nice to be honest, but he's throwing his toys out of his pram about it.

**Comment by Crissov** [comment]
Related: #5927 (Asian zodiac). I have posted canonic images (as emojis) there, including Unicode code points. JFTR, here are the standard Western symbols:

1. ♈ U+2648 Aries
2. ♉ U+2649 Taurus
3. ♊ U+264A Gemini
4. ♋ U+264B Cancer
5. ♌ U+264C Leo
6. ♍ U+264D Virgo
7. ♎ U+264E Libra
8. ♏ U+264F Scorpius
9. ♐ U+2650 Sagittarius
10. ♑ U+2651 Capricorn
11. ♒ U+2652 Aquarius
12. ♓ U+2653 Pisces

**Comment by Ladiczek** [comment]
!!! Thanks a lot" Havent any idea about this!

**Comment by solsystem** [comment]
So font awesome 5 came out, and still no astrology icons and a definitely not a complete set of planet symbols. I gave up and made my own font-icon pack.

**Comment by enfascination** [comment]
+1

**Comment by TheOldMan2000** [comment]
Hi all,

Hope you don't mind but I'm adding a comment as well as a thumbs-up!
FA are doing a brilliant job with FA5 but I would love to see either the zodiac signs added or indeed a full  astrology pack launched with planet/moon/elemental signs etc. 

Certainly the recognised zodiac birth-signs would be used in social profiles a lot, after all there are almost 7.8 billion of us at the moment! 

Many thanks.

**Comment by Cow258** [comment]
+1

**Comment by TheOldMan2000** [comment]
Come on FA! You still don't have any Astrology Birth-sign icons? Ridiculous.

**Comment by tec-lab** [comment]
+1

**Comment by mikeher** [comment]
+1

**Comment by Diesouto** [comment]
+1

**Comment by leecollings** [comment]
> What's the news with this? Doesn't look like anything happened, and it's been over a year....

Make that 7 years now!!!!!!!!!!!! WHAT ON EARTH???

**Comment by leecollings** [comment]
> I also need to clarify that there is nothing against this specific request, which is open, labeled and treated like [all other requests](https://github.com/FortAwesome/Font-Awesome/issues?q=is%3Aopen+is%3Aissue+label%3A%22new+icon%22+sort%3Acomments-desc)

Hurry up and do this already then!

**Comment by TheOldMan2000** [comment]
As much as I'd love to have seen the rather measly number of 12 Zodiac icons produced, I've become so disillusioned with FA since FA 6 was announced. 

Icon requests have sadly become a tragedy. I was once happy to invest in FA, but now expecting hobby, personal and freelance site owners to pay 100 bucks a year for some icons is absolutely ludicrous. Save that malarkey for the big corporate entities, not the little guys.

Factor in when you search for existing open icon requests, it tells there are none, so you make a request following the requested protocol, then some rude moderator publicly fish slaps you for not checking if a request exists. 

Forgive my expression of frustration, but I won't be holding my breath for any Zodiac or Astrology icons any time soon.

**Comment by DavidMichaelangelo** [comment]
How do you downvote icon requests? 
Surely there are more important icon sets than signs relating to snake oil mysticism.

**Comment by mikeher** [comment]
> How do you downvote icon requests? Surely there are more important icon sets than signs relating to snake oil mysticism.

Remember that It's not necessarily the people that request the icons that believe in snake oil mysticism. Their clients might, and clients come from all the circles of hell.

**Comment by DavidMichaelangelo** [comment]
> > How do you downvote icon requests? Surely there are more important icon sets than signs relating to snake oil mysticism.
> 
> Remember that It's not necessarily the people that request the icons that believe in snake oil mysticism. Their clients might, and clients come from all the circles of hell.

Those poor souls

**Comment by reformed** [comment]
Just tell those clients to knock it off with the weird superstitious nonsense.

**Comment by TheOldMan2000** [comment]
Some childish hater comments here!

People have all sorts of genuine and valid needs for requesting astrology icons, aside from clients and websites needing or requesting them. Nobody is seeking you to believe in the pseudo science, after all interest in the stars and alignment has been around since the 2nd millennium BC.

When you look at some of the themed icons already released, there are a lot more that could be viewed equally or as far less valid or important. Flying spaghetti monsters, Star Wars religious book icons anyone? 

I love Star Wars and pasta, I'm just saying.

**Comment by leecollings-sb** [comment]
I think it is disgraceful that this request still hasn't been granted, given how ridiculous some of the current icons are.

It goes to show how stubborn the creators are being and just not listening to popular requests and pleads from the people actually using their products.

Several years later and still waiting!

**Comment by reformed** [comment]
> I think it is disgraceful that this request still hasn't been granted, given how ridiculous some of the current icons are.
> 
> It goes to show how stubborn the creators are being and just not listening to popular requests and pleads from the people actually using their products.
> 
> Several years later and still waiting!

How do you know the motive of the creators is rooted in stubbornness? What if they're just too busy? Shouldn't we give someone the benefit of the doubt? Maybe they should add a crybaby icon..

**Comment by Diesouto** [comment]
This request was made in 2014, too busy? are you serious? hahaha


> How do you know the motive of the creators is rooted in stubbornness? What if they're just too busy? Shouldn't we give someone the benefit of the doubt? Maybe they should add a crybaby icon..

**Comment by teclabat** [comment]
Some of you don't know how open source works. This is free software without any charge. You can politely make a request and thats it. You are free to clone this repository and extend it with whatever you want.

**Comment by mikeher** [comment]
> > I think it is disgraceful that this request still hasn't been granted, given how ridiculous some of the current icons are.
> > It goes to show how stubborn the creators are being and just not listening to popular requests and pleads from the people actually using their products.
> > Several years later and still waiting!
> 
> How do you know the motive of the creators is rooted in stubbornness? What if they're just too busy? Shouldn't we give someone the benefit of the doubt? Maybe they should add a crybaby icon..

Obviously we are all grateful for their work. The fact that they provide a free solution does not exempt them from criticism though. @leecollings-sb makes a good point. Who tf needs a spaghetti monster?!

@reformed The fact that you sarcastically suggests that they should have a crybaby icon kinda works against you since they have both multiple cry emojis and even a baby icon.

I think it's fair to assume that there are some opinionated developers at play here. "Luckily" it seems that the company is gradually turning more profit-focused so it probably won't be long before those opinions are overruled by customer desires.

**Comment by jbnv** [comment]
The project being open source is great if you want an icon AND you know how to make them. I personally have limited experience in making SVGs. So all I can do at this point is make requests and upvote requests that I support. I have significant questions about the priorities of the FA team, but if FA has a flying spaghetti monster because someone wanted one and made it themselves, we can't pin that on the team. 

The reality of software development, even in open source, is that nobody works for free. Even work we do for ourselves as hobbies we do because the reward is something we can use, that adds value to our lives. Somewhere on the Internet, someone derived value from making a flying spaghetti monster icon, but that same person has no interest in making zodiac icons. C'est la vie. Now, what can we do to give those who can make icons incentive to make the icons we want? Usually that answer is to pay them. That's just reality.

**Comment by reformed** [comment]
Well then with the cry emojis and baby icon each of you are already adequately represented by the available icons. Now you're just being ungrateful. It is entirely within Fort Awesome's prerogative to make one icon or set of icons and not another, for whatever reason (or for no reason). If you don't like it, keep crying about it or start your own company and make your own. But for you to presume their motive is unjust and unacceptable; it just demonstrates your immaturity (hence the crybaby snipe).

**Comment by tagliala** [comment]
Hi everyone, thanks for your interest in this request, which is at the first post of our [Leaderboard](https://fontawesome.com/community/leaderboard/new)!

I'm going to remove some comments

You may also be interested in taking a look at our [Icon Leaderboard FAQs](https://fontawesome.com/community/leaderboard/new#faqs), they contain a lot of information about how requests work and why there are some requests that seem unaddressed

Please also keep in mind that this is a request for **60** new icons (12 icons in 5 styles: solid, regular, light, thin, duotone)

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
